cli/azd/pkg/tools/github/github.go

```diff
@@ -51,7 +51,7 @@ func NewGitHubCli(ctx context.Context, console input.Console, commandRunner exec
 
 // cGitHubCliVersion is the minimum version of GitHub cli that we require (and the one we fetch when we fetch bicep on
 // behalf of a user).
-var cGitHubCliVersion semver.Version = semver.MustParse("2.22.1")
+var cGitHubCliVersion semver.Version = semver.MustParse("2.28.0")
```

Comment:
FYI - In #1975 I renamed this to be GitHubCliVersion and exported it so I could reference it from `github_provider_test.go`

Reply:
Should we create a Getter() in those cases?   to keep encapsulation and prevent updating the value?